tools/cross/format.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import os
+import re
+import subprocess
+import logging
+import typing
```

Comment:
Nit: you should use `from` imports for type hints and such:
```suggestion
from argparse import ArgumentParser, Namespace
from typing import List, Optional
```